spacialgaze-plugins/ssb.js

```diff
@@ -466,7 +471,7 @@ class SSB {
 		return true;
 	}
 	activate() {
-		if (this.species && (this.movepool.length > 0 || this.cMove) && this.ability) {
+		if (this.species && (this.movepool.length > 0 || this.cMove) && (this.ability && ability.id !== 'shadowtag')) {
```

Comment:
Maybe use || instead of && (and add the other banned abilities)

Reply:
Why are you adding a validation check to `activate()` anyways? True validation is handled elsewhere (idr the exact function name, but its called by `/ssb validate`)

Reply:
I know i shouldn’t >_>

It’s just wobbuffet was being stubborn